packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -273,9 +279,13 @@ describe('UIForm component', () => {
 			wrapper.instance().onSubmit(submitEvent);
 
 			// then
-			expect(props.setErrors).toBeCalledWith(submitEvent, {
-				firstname: 'Missing required field',
-			});
+			expect(props.setErrors).toBeCalledWith(
+				submitEvent,
+				{
+					firstname: 'Missing required field',
+				},
+				wrapper.instance().focusFirstError,
```

Comment:
Replace that to `expect.anything()`.
This is not the goal of this test, and this 3rd arg is tested in a specific test

Reply:
done